src/css/skins/five.less

```diff
@@ -159,6 +157,17 @@
         }
     }
 
+    .jw-icon-cc {
```

Comment:
`.jw-icon-cc` should be `jw-icon-tooltip` so that all menus (hd and audiotracks) also get these colors right?